Informative error when the entry is missing a `bump: `

Just bumping this. Empty changelog_entry.yaml files produce this message:
```
build-changelog changelog.yaml --output changelog.yaml --update-last-date --start-from 0.0.1 --append-file changelog_entry.yaml
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.9.16/x64/bin/build-changelog", line 8, in <module>
    sys.exit(main())
  File "/opt/hostedtoolcache/Python/3.9.16/x64/lib/python3.9/site-packages/yaml_changelog/build.py", line 267, in main
    cl = Changelog(
  File "/opt/hostedtoolcache/Python/3.9.[16](https://github.com/PolicyEngine/policyengine-us/actions/runs/4475331706/jobs/7864646342#step:4:17)/x64/lib/python3.9/site-packages/yaml_changelog/build.py", line 82, in __init__
    self._parse_from_yaml(self.file, append)
  File "/opt/hostedtoolcache/Python/3.9.16/x64/lib/python3.9/site-packages/yaml_changelog/build.py", line 105, in _parse_from_yaml
    self.entries.extend(yaml.safe_load(f))
TypeError: 'NoneType' object is not iterable
make: *** [Makefile:[20](https://github.com/PolicyEngine/policyengine-us/actions/runs/4475331706/jobs/7864646342#step:4:21): changelog] Error 1
```
With more new volunteers, it'd be helpful for them to know what to do.

--- a/Makefile
+++ b/Makefile
@@ -6,5 +6,4 @@ install:
 	pip install -e .
 
 format:
-	autopep8 -r . -i
 	black . -l 79

--- a/yaml_changelog/build.py
+++ b/yaml_changelog/build.py
@@ -100,9 +100,22 @@ class Changelog:
     def _parse_from_yaml(self, file: Path, appended_file: Path = None):
         with open(file) as f:
             self.entries = yaml.safe_load(f)
+        for entry in self.entries:
+            for change_type in CHANGE_TYPES:
+                # Move insite entry["changes"]
+                if change_type in entry:
+                    if "changes" not in entry:
+                        entry["changes"] = {}
+                    entry["changes"][change_type] = entry[change_type]
+                    del entry[change_type]
         if appended_file is not None:
             with open(appended_file) as f:
-                self.entries.extend(yaml.safe_load(f))
+                try:
+                    self.entries.extend(yaml.safe_load(f))
+                except TypeError:
+                    raise ValueError(
+                        f"You haven't provided a changelog entry in {appended_file}. It should look like this: \n\n- bump: minor\n  added:\n  - Some new feature."
+                    )
 
     def _parse_from_md(self, file: Path):
         with open(file) as f:

